clusterloader2/README.md

```diff
@@ -63,8 +79,21 @@ This measurement gathers the memory profile provided by pprof for a given compon
 - **MetricsForE2E** \
 The measurement gathers metrics from kube-apiserver, controller manager,
 scheduler and optionally all kubelets.
+- **PodStartupLatency** \
+This measurement observes pods being created and collects is startup times. \
+This measurement is a SLO. If given startup time threshold is violated, the error is returned.
```

Comment:
This measurement verifies if pod startup SLO is satisfied.

Reply:
Done.